src/Plugin/jsonapi/FieldEnhancer/CardLinkEnhancer.php

```diff
@@ -84,8 +84,13 @@ public function getAlias($nid) {
    *   The array of fields value.
    */
   public function getCardFields($nid) {
```

Comment:
Hi @MdNadimHossain  instead of passing an node id to the functions you could pass a node entity.